website/source/partials/builder/vmware/vmx/_Config-not-required.html.md

```diff
@@ -14,11 +14,6 @@
     scenarios. Most users will wish to create a full clone instead. Defaults
     to false.
     
--   `remote_type` (string) - The type of remote machine that will be used to
```

Comment:
Because it was removed here: https://github.com/hashicorp/packer/pull/8232#discussion_r335216752 it got removed from the docs. 

But if you look carefully it's there twice in the packer docs https://www.packer.io/docs/builders/vmware-vmx.html#remote_type